.github/workflows/testrunner.yml

```
@@ -3,7 +3,7 @@ name: Testrunner
 on:
   push:
     branches:
-      - '1.x'
+      - '2.x'
```

We shouldn't generate a testrunner image when a PR is created. Maybe this would work better?

```suggestion
      - '2.x'
    paths:
      - 'docker/**'
```

cc @emmettbutler